Makefile

```diff
@@ -0,0 +1,20 @@
+.PHONY: build up 
+
+# Create directories for mounting volumes
+
+all: build up
+
+# Build the Docker images
+build: 
+	docker compose -f docker-compose.yml build
+
+# Start the services
+up: build	
+	docker compose -f docker-compose.yml up -d
```

Comment:
Extracted from the subject ->

• Everything must be launched with a single command line to run an autonomous container provided by Docker . Example : docker-compose up --build

shall we combine build and up?